### PR TITLE
VTU FSR outputs are now organized by domain into folders.

### PR DESCRIPTION
**Pull request by zdodson94**
Small change made for organizing VTU outputs into folders.

### CODE REVIEW

**Comment by zdodson94** [comment]
Changes were also made to master for this task, but I am waiting for this to be merged first.

**Comment by bkochuna** [comment]
Yeah. I had t talked to Zack about that yet, I've been thinking about folder name, but this part won't change, this is what gets written into the pvtu file. In MPACT is where we'll have to make the changes for a top level directory.

I was thinking of calling that folder fsr_<casename>

**Comment by bscollin** [comment]
As long as I don't have 1000 folders in my directory, I"m good :)

**Comment by bkochuna** [comment]
One other comment Zack, there is a routine in IO_strings called SlashRep that handles the different slashes for Windows and linux. This will let you remove the ifdef's in your changes.

**Comment by bkochuna** [comment]
Ben, can you accept the changes so we can merge? (or do you mind if I just dismiss your review?)

File: src/VTUFiles.f90 [hunk 1]
```diff
@@ -472,17 +473,19 @@ SUBROUTINE close_VTUXMLFileType(myVTKFile)
...
       CHARACTER(LEN=128) :: sint
       INTEGER(SIK),INTENT(IN) :: funit,procs,rank
       INTEGER(SIK) :: i,j,iord
       CHARACTER(LEN=128) :: fname,fmtStr
       !
       IF((procs>1).AND.rank == 0) THEN
-        OPEN(unit=funit,file=TRIM(filen)//'.pvtu')
+        sint='fsr_'//TRIM(case)//'/'
+        CALL SlashRep(sint)
+        OPEN(unit=funit,file=TRIM(sint)//TRIM(filen)//'.pvtu')
```
Comment (bscollin) [inline src/VTUFiles.f90]: This isn't what I was thinking... but I'm okay with it.  I figured the pvtu files would be in your main repo, everything else would be in the subdirectory.  But again, I don't really care that much.

File: src/VTUFiles.f90 [hunk 2]
```diff
@@ -472,17 +473,19 @@ SUBROUTINE close_VTUXMLFileType(myVTKFile)
...
-      CHARACTER(LEN=*),INTENT(IN) :: filen
+      CHARACTER(LEN=*),INTENT(IN) :: case,filen
       CHARACTER(LEN=128) :: sint
       INTEGER(SIK),INTENT(IN) :: funit,procs,rank
       INTEGER(SIK) :: i,j,iord
       CHARACTER(LEN=128) :: fname,fmtStr
       !
       IF((procs>1).AND.rank == 0) THEN
-        OPEN(unit=funit,file=TRIM(filen)//'.pvtu')
+        sint='fsr_'//TRIM(case)//'/'
```
Comment (bscollin) [inline src/VTUFiles.f90]: Maybe a little picky... but if futility is supposed to be a generic library, should we hard-code fsr into the name?  Its pretty MPACT specific.

File: src/VTUFiles.f90 (hunk 1 above)
Comment (zdodson94) [inline src/VTUFiles.f90]: I just figured it was cleaner to put everything in a subdirectory.

File: src/VTUFiles.f90 (hunk 2 above)
Comment (zdodson94) [inline src/VTUFiles.f90]: I could change it to vtu_<casename>. @bkochuna, what do you think?

File: src/VTUFiles.f90 (hunk 1 above)
Comment (bscollin) [inline src/VTUFiles.f90]: Like I said... both of those comments are pretty minor.  Just easier to have the discussion now.

Comment (bkochuna) [inline src/VTUFiles.f90]: Its basically:
 - fsr_\<casename\>
   - pvtu files
   - DOMAIN_XX
      - vtu files

**Pull request merged**